docs/UserGuide.md

```diff
@@ -58,105 +102,154 @@ management tasks done faster than traditional GUI apps.
-  be `2020-10-11`
-  if the existing quantity and expiry date of the 1st item are both not `10` and `2020-10-11` respectively.
-* `edit 2 n/Chocolate Bread t/` Edits the name of the 2nd item to be `Chocolate Bread` and clears all existing tags if
+* `edit 1 q/10 e/2020-10-11` edits the quantity of the 1st item to be `10` and expiry date of the 1st item to
+  be `2020-10-11` if the existing quantity and expiry date of the 1st item are both not `10` and `2020-10-11` respectively.
+* `edit 2 n/Chocolate Bread t/` edits the name of the 2nd item to be `Chocolate Bread` and clears all existing tags if
   there are existing tags and/or existing name of the 2nd item is not `Chocolate Bread`.
 
-### Locating items by name: `find`
+### 3.4. Deleting an item : `delete`
 
-Finds items whose names contain any of the given keywords, either in full or partially.
+Looking to get rid of an existing item from the inventory? Use this command.
 
-* The search is case-insensitive. e.g. `milk` will match `Milk`
-* The order of the keywords does not matter. e.g. `Bread Chocolate` will match `Chocolate Bread`
-* Only the item name is searched.
-* Items matching at least one keyword will be returned (i.e. `OR` search). e.g. `Chocolate Milk` will
-  return `Cadbury Chocolate`, `Almond Milk`
+Format: `delete INDEX`
 
-Format 1: `find KEYWORD [MORE_KEYWORDS]`
+<div markdown="block" class="alert alert-info">
 
-* Only full words will be matched e.g. `Chocolate` will not match `Chocolates`
+**:information_source: Notes about the delete command:**<br>
 
-Format 2: `find */KEYWORD [MORE_KEYWORDS]`
+* Deletes the item at the specified `INDEX`.
+* The index refers to the index number shown in the currently displayed item list.
+* The index **must be a positive integer** 1, 2, 3, …​
 
-* Partial words will be matched e.g. `Choco` will match `Chocolates`
+</div>
 
 Examples:
+* `list` followed by `delete 2` deletes the second item in the entire inventory.
+* `find Chocolate` followed by `delete 1` deletes the first item in the result of the `find` command.
+* `list Room 2` followed by `delete 3` deletes the third item in the recorded list of items in Room 2.
 
-* `find Chocolate` returns `chocolate` and `Chocolate Milk`
-* `find potato chip` returns `Potato Biscuit` and `chocolate chip`
-* `find */Burger` returns `CheeseBurger` and `fishburger`
-* `find */cheese egg` returns `MacAndCheese` and `eggs`
 
-### Deleting an item : `delete`
+### 3.5. Listing items : `list`
 
-Deletes the specified item from the inventory.
+Want to view all your items? What about items at a specific location or with a specific tag? This command helps you to 
+do just that!
 
-Format: `delete INDEX`
+You can use this command in 3 different ways.
 
-* Deletes the item at the specified `INDEX`.
-* The index refers to the index number shown in the displayed item list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* #### 3.5.1. Listing all items in StoreMando 
```

Comment:
Why is this not bold anymore? How will it look like in the user guide page?

Reply:
The subheaders will appear in bold when displayed in HTML so it will still be fine.